### DISCUSSION TITLE
how to use GLSL shaders on OpenFL

### DISCUSSION THREAD

**Discussion by Fadeyauso** [HaxeFlixel/flixel#2472] [Q&A]
like i'm taking shaders from shadertoy and they don't work

**Comment by Geokureli** [comment]
openfl uses GLSL2.0 shadertoy uses 3.0 which has more features

**Reply by Geokureli** [reply]
Here's an example of a GLSL 2.0 version of a shader I made
https://github.com/Geokureli/Advent2020/blob/master/source/vfx/DitherShader.hx
and here's the early prototype I made in shadertoy, first
https://www.shadertoy.com/view/tsyfzD

I hope this helps
